apps/masterbots.ai/app/share/[id]/page.tsx

```
@@ -1,48 +1,48 @@
 
-  return {
-    title: chat?.title.slice(0, 50) ?? 'Chat'
-  }
+	return {
+		title: chat?.title.slice(0, 50) ?? 'Chat',
+	}
 }
 
 export default async function SharePage({ params }: SharePageProps) {
-  const chat: any = { sharePath: '', title: '' } //await getSharedChat(params.id)
+	const chat: any = { sharePath: '', title: '' } //await getSharedChat(params.id)
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)